demos/tutorial/alignment/scoring_scheme_affine_dgs.cpp

```diff
@@ -0,0 +1,42 @@
+//![main]
+#include <iostream>
+#include <seqan/align.h>
+
+using namespace seqan;
+
+int main()
+{
+    typedef String<AminoAcid> TSequence;                 // sequence type
+    typedef Align<TSequence, ArrayGaps> TAlign;      // align type
```

Comment:
I don't want to be stringent about this.
But I think indentations are important since it's a tutorial.

Can you consider this? 
http://seqan.readthedocs.org/en/master/StyleGuide/Cpp.html#inline-comments